Core/wbDefinitionsCommon.pas

```diff
@@ -955,6 +961,21 @@ procedure wbATANsAfterSet(const aElement: IwbElement; const aOldValue, aNewValue
   wbCounterAfterSet('ATCP - Activity Count', aElement);
 end;
 
+procedure wbBODSsAfterSet(const aElement: IwbElement; const aOldValue, aNewValue: Variant);
+begin
+  wbCounterAfterSet('BODC - Count', aElement);
+end;
+
+procedure wbBODCsAfterSet(const aElement: IwbElement; const aOldValue, aNewValue: Variant);
+begin
+  wbCounterAfterSet('BODM - Count', aElement);
+end;
+
+procedure wbEXASsAfterSet(const aElement: IwbElement; const aOldValue, aNewValue: Variant);
+begin
+  wbCounterAfterSet('EXAC - Count', aElement);
+end;
+
```

Comment:
I'll have to think about a better way to do this than defining countless callbacks... should be possible to specify simply the signature of the counter subrecord on the RArray definition...